wgsl/index.bs

```diff
@@ -368,38 +368,43 @@ However, UTF-8 is always a valid encoding for a WGSL program.
 Note: The intent of promoting UTF-8 like this is to simplify interchange of WGSL programs
 and to encourage interoperability among tools.
 
-WGSL program text consists of a sequence of characters, grouped into contiguous non-empty sets forming:
+WGSL program text consists of a sequence of code points, grouped into contiguous non-empty sets forming:
 * [=comments=]
 * [=tokens=]
 * [=blankspace=]
 
-The program text must not include a null character.
+The program text must not include a null code point (`U+0000`).
 
-<dfn>Blankspace</dfn> is any combination of one or more of the following characters:
-* space
-* horizontal tab
-* linefeed
-* vertical tab
-* formfeed
-* carriage return
+<dfn>Blankspace</dfn> is any combination of one or more of the following code points:
+* space (`U+0020`)
+* horizontal tab (`U+0009`)
+* line feed (`U+000A`)
+* vertical tab (`U+000B`)
+* form feed (`U+000C`)
+* carriage return (`U+000D`)
+* next line (`U+0085`)
+* left-to-right mark (`U+200E`)
+* right-to-left mark (`U+200F`)
```

Comment:
We should probably be referencing default-ignorable characters, rather than listing some bidi characters but not others.

Reply:
Please see https://github.com/gpuweb/gpuweb/pull/2734#discussion_r847914766